src/data/firefox-health-dashboard.yaml

```diff
@@ -0,0 +1,30 @@
+name: Firefox Health Dashboard
+incon: https://codetribute.mozilla.org/projects/reporting
+summary: Show statistics and aggregates for monitoring current projects
+introduction: |
+  ## About Firefox Health Dashboard
+
+  A JSX/React app that pulls data from multiple sources and presents charts and statistics for Mozilla development teams
+
+  ## Who Works on the Heath Dashboard?
+
+  A collection of friendly Mozilla engineers and contributors
+
+  ## How Do I Get Started?
+
+  The [main Github page](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/#developing)
+  has instructions on how to begin. You will require `node` and `yarn` installed.
+
+  ### How Do I Write the Code?
+
+  1. [Find an Issue to work on](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues) or
+  [looks at the site](https://health.graphics/) and see if there is something small you would like to fix.
+  2. Fork-and-clone the repo, make some change, and submit a PR. Someone will be happy to guide you
+  through the process of coding and getting it acepted.
+
+  ## How Do I Get Help?
+
+  Anyone on `#frontend-infra` on `irc.mozilla.com` can answer your questions.
```

Comment:
`s/irc/irccloud/`.

Reply:
I think a link to https://wiki.mozilla.org/IRC might be beneficial to properly set up IRC. In `taskcluster.yml` we wrote:

> We are also available on [irc](https://wiki.mozilla.org/IRC) in the `#taskcluster-contributors` channel.